step-capstone/src/components/TravelObjects/TravelObject.js

```diff
@@ -12,10 +12,8 @@ export default function TravelObject(props) {
     
     const handleClick = (event) => {
         setAnchorEl(event.currentTarget);
-        // if has a placeId, zoom to marker on map
```

Comment:
Curious: What made this check no longer necessary?

Reply:
I have a check in the map component that only zooms to the marker if it exists in the map (key is placeId)

Reply:
Actually never mind, I did need it--thanks for catching!